specs/bep-v1.rst

```diff
@@ -844,6 +862,140 @@ XDR
         int Code;
     }
 
+DownloadProgress (Type = 8)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The DownloadProgress message is used to notify remote devices about partial
+availability of files. DownloadProgress messages are only sent when progress 
+has been made or the state of the file being downloaded has changed. By default,
+these messages are sent every 5 seconds Each DownloadProgress message is for 
+a specific folder and MAY contain zero or more FileDownloadProgressUpdate 
```

Comment:
Nothing MAY about it, it has no choice but to contain zero or more FileDownloadProgressUpdates :)